.eslintrc

```diff
@@ -0,0 +1,75 @@
+{
+  "env": {
+    "node": true,
+    "builtin": true
+  },
+  "rules": {
+    "block-scoped-var": 2,
```

Comment:
I understand ```2``` is an equivalent of ```error```. I wonder if a consistent value should be used across all our projects to reduce a possible confusion. If agreed, I would vote on using ```error```.